src/content/1.7/project/maintainers-guide/summary-github-lifecycle.md

```diff
@@ -56,6 +56,84 @@ We sometimes receive support requests on GitHub, ranging from questions about th
 
 There are some Issues which do not fit in the previous categories, such as [releases Issues](https://github.com/PrestaShop/PrestaShop/issues/20804). They serve a specific purpose.
 
+## Projects
+
+Available in the ‘[Projects](https://github.com/PrestaShop/PrestaShop/projects)’ tab of the PrestaShop’s GitHub repository, versions kanbans organize the Core team’s work. Both the product and the tech teams use it to prepare and plan the next developments and, since we deal with an open source project, the purpose of having those kanbans public is to help the community and the external mainteners to know what is at stake.
```

Comment:
Kanbans are not meant to organize the Core team's work, else there would only be a single "Core kanban" and we'd only work on what's in there. Kanbans are there first and foremost to define the scope of a minor version in terms of _what the Core and Product teams agree **must be included**_ before that release is out. This explains why we don't add community "nice to haves" unless we mean to see those through ourselves.

Reply:
So, we should keep in Kanban only the must-have of the version, no matter if they are done by a developer working for PrestaShop Corp or by a community member.
In this case how do we manage "nice to have" ? In another Kanban ?

Reply:
> In this case how do we manage "nice to have" ? In another Kanban ?

Could be. The deeper question is what exactly do we intend to track on the Kanban. One option would be to set up a kanban for each individual version, down to the patch version. In that case, any issue added to a given kanban would _de facto_ become a "Must have" for that specific version. "Nice to haves" would cease to exist in terms that everything is a must have until it's either done or removed from the kanban. When the kanban is "all done", then the version is ready for release and the kanban can be closed.

Reply:
I edited this part, see https://github.com/PrestaShop/docs/pull/807/commits/9ea846234d916f584ce10b38a4fa8341910a6bf9